DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
-                 #                    version => version of the template
-                 #                    uri => url of the template
-email.uri        # => URL of the notification
+```ruby
+personalisation: {
+  name: "John Smith",
+  year: "2016"                      
+}
 ```
 
-Otherwise the client will raise a `Notifications::Client::RequestError`:
+You can leave out this argument if a template does not have any placeholder fields for personalised information.
 
-|`error.code`|`error.message`|
-|:---|:---|
-|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM of 10 requests per 10 seconds"`<br>`}]`|
-|`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient using a team-only API key"`<br>`]}`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|
+#### reference (optional)
 
-</details>
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
+```ruby
+reference: "your_reference_string";
+```
 
-#### Arguments
+You can leave out this argument if you do not have a `reference`.
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+#### email_reply_to_id (optional)
 
-##### `email_address`
-The email address of the recipient, only required for email notifications.
+This is an email reply-to address specified by you to receive replies from your users. Your service cannot go live until you set up at least one of these email addresses. To set up:
 
-##### `template_id`
+1. Sign into your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you need to change to another service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the Email section and select __Manage__ on the __Email reply to addresses__ row.
+1. Select __Change__ to specify the email address to receive replies, and select __Save__.
 
-Find by clicking **API info** for the template you want to send.
+```ruby
+email_reply_to_id: '8e222534-7f05-4972-86e3-17c5d9f894e2'
+```
 
-##### `reference`
+You can leave out this argument if you do not have have an `email_reply_to_id`.
 
-An optional identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+### Response
 
-You can omit this argument if you do not require a reference for the notification.
+If the request to the client is successful, the client will return a `Notifications::Client:ResponseNotification`:
 
-##### `email_reply_to_id`
+```ruby
+ @content={"notification content"},
```

Comment:
The hash containing the content looks like this
`{"body"=>"The body of the email", "from_email"=>"sender@something.com", "subject"=>"The email subject"}`